xz/xz.cabal

```diff
@@ -0,0 +1,77 @@
+cabal-version:       2.2
+name:                xz
+version:             5.2.5.5
+
+synopsis:            LZMA/XZ compression and decompression
+homepage:            https://github.com/hasufell/lzma-static
+bug-reports:         https://github.com/hasufell/lzma-static/issues
+license:             BSD-3-Clause
+license-file:        LICENSE
+author:              Julian Ospald
+maintainer:          hasufell@posteo.de
+copyright:           (c) 2015, Herbert Valerio Riedel, (c) 2021 Julian Ospald
+stability:           experimental
+category:            Codec, Compression
+build-type:          Simple
+tested-with:         GHC==8.6.5, GHC==8.8.4, GHC==8.10.4
+description:
+    This package provides a pure interface for compressing and
+    decompressing
+    <https://en.wikipedia.org/wiki/LZMA LZMA (Lempel–Ziv–Markov chain algorithm)>
+    streams of data represented as lazy @ByteString@s. A
+    monadic incremental interface is provided as well. This package
+    relies on the <http://tukaani.org/xz/ liblzma C library>.
+
+extra-doc-files:
+  Changelog.md
+
+source-repository head
+  type:     git
+  location: https://github.com/hasufell/lzma-static.git
```

Comment:
add `subdir:` in both cabal..

I don't see any reason to change the repo name. so the location could remain same